src/main/resources/com/turo/pushy/console/pushy-console_de.properties

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2018 Turo Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+alert.bad-certificate.title=Ungültiges Zertifikat
+alert.bad-certificate.header=Das ausgewählte Zertifikat ist kein gültiges APNs Zertifikat.
+alert.bad-certificate.content-text=Die UID im Subject Feld des Zertiifikates ist kein gültiges 'APNs topic'.
+alert.notification-failed.title=Senden der Push-Benachrichtigung fehlgeschlagen
+alert.notification-failed.header=Beim Senden der Push-Benachrichtigung ist ein Fehler aufgetreten.
+certificate-chooser.filter.pkcs8_and_pkcs12=PKCS#8 und PKCS#12 Dateien
+certificate-chooser.filter.pkcs12=PKCS#12 Dateien (Zertifikate)
+certificate-chooser.filter.pkcs8=PKCS#8 Dateien (Authentifizierungs-Token)
+certificate-chooser.filter.all=Alle Dateien
+certificate-password-dialog.title=Kennwort eingeben
+certificate-password-dialog.header=Bitte geben Sie das Kennwort für {0} ein.
+certificate-password-dialog.prompt=Kennwort
+delivery-priority.immediate=Sofort
+delivery-priority.conserve-power=Stromsparend
+notification-result.placeholder=Keine Push-Benachrichtigung gesendet
+notification-result.details.accepted=n.a.
+notification-result.details.expiration={0} ({1,date,dd-MM-yyy} {1,time,HH:mm:ss})
+notification-result.status.accepted=Angenommen
+notification-result.status.rejected=Zurückgewiesen
+password-dialog.incorrect-password=Das eingegebene Kennwort ist nicht korrekt.
+pushy-console.title=Pushy Console
+fxml.apns-server.label=APNs Server
+fxml.port.label=Port
+fxml.credentials.label=Token / Zertifikat
```

Comment:
Can you say more about this (if only for my own curiosity about German)? It seems like we're going from a broad idea ("credentials") to an enumeration of the types of credentials we'll accept ("token or certificate").

Reply:
While the translation of 'credential' as 'Berechtigungsnachweis' is formally correct, 'Berechtigungsnachweis' is not widely used and users will be surprised to select a file with cryptographic credentials. 'Berechtigungsnachweis' has a very broad meaning which ranges from cryptographic token to a signed paper document. And in this particularly case 'Token/Zertifikat' is more accurate what is expected from the user.
 
Unfortunately Apple does not provide a german documentation or glossary where you can lookup the 'official' term.

Reply:
Understood. Thanks!